test/appium/tests/atomic/account_management/test_profile.py

```diff
@@ -223,7 +226,7 @@ def test_need_help_section(self):
         base_web_view.find_full_text('Frequently Asked Questions')
         base_web_view.click_system_back_button()
         profile_view.submit_bug_button.click()
-        profile_view.find_full_text('#status')
+        profile_view.wait_for_element_starts_with_text('Welcome to Gmail')
```

Comment:
Here we need to update not the check but above button (on to 'Request a feature') which redirects to '#status' channel.
For the submit_bug_button (old red 'Report bug' button) we have separate test case (which should be updated as well in TestRail btw)

Reply:
you are right, fixed